config/index.js

```diff
@@ -63,6 +63,28 @@ bookshelf.plugin('virtuals');
 bookshelf.plugin('pagination');
 config.bookshelf = bookshelf;
 
+// Overwrite node-pg-types date parser. See https://github.com/tgriesser/knex/issues/1750
+const pgTypes = require('pg').types;
+
+function parseDate(val) {
+  return new Date(Date.parse(val));
```

Comment:
Is it possible for `parseDate()` to be called with null values as well? If so, it needs to handle this case, similarly to `parseDateArray`